block-node/test-specifications/local-persistence/block-as-local-file-persistence-tp.md

```diff
@@ -0,0 +1,85 @@
+| E2ETC_BLFP_0008 | `Verify Cleanup of Unverified Root at System Startup`                              | `Block` files under `unverified` that may exist during system startup must be all deleted as they are deemed untrustworthy. It is not possible to assert the state thereof.                                                                                                                                                                                                                                                                                                    | Any `Block` files persisted under the `unverified` root must be deleted during system startup. The `unverified` root must be completely empty by the time the system is up and running, ready to operate.                                                                                                                                                                                                                                                                                                                                                    | Ensure that an arbitrary amount of `Block` files are present in the `unverified` root, then proceed to startup the system.                                                                                                                                                                       | `unverified` root is completely empty by the time the system is started and ready to operate.                                                                                                                                                                                                                                                            |         N         |
```

Comment:
Why would it not be possible to verify a block file under `/unverified` after startup?
Shouldn't the block proof process cover this?

For recovery reasons did we consider rather checking if a file in `/unverified` is not in `/live` before cleaning up in this case?